strapi/strapi
# feat(main-nav): Main nav refactoring, refactor the main nav container and all the links and user profile

What does it do?
It shows the new main nav. Now all the links are shown with just the icon, the divider between the links are removed and we are using new components in the admin to render the Main nav

Why is it needed?
In the refactoring of the Main Nav we want a new narrow nav to add more space for the important content, also we removed the Dividers (hr elements) between the links

How to test it?
Just check the new main nav in the different themes, using a CE license and EE license, all the states of the links (active, hover) and the Profile button

You can find all the info and link to the figma file in the Jira ticket

Related issue(s)/PR(s)
CS-758 and CS-766
